### DISCUSSION TITLE
Adding Wazuh agents.

### DISCUSSION THREAD

**Discussion by shapesh** [Security-Onion-Solutions/securityonion#1693] [Unsupported Versions] [answered]
Am trying to add wazuh agents but am getting just to options unlike the previous version 16.
[root@securityonion ~]# sudo /var/ossec/bin/manage_agents

****************************************
* Wazuh v3.13.1 Agent manager.         *
* The following options are available: *
****************************************
   (I)mport key from the server (I).
   (Q)uit.
Choose your action: I or Q:

**Comment by dougburks** [comment, marked as answer]
Have you tried the following?
```
sudo docker exec -it so-wazuh /var/ossec/bin/manage_agents
```

For more information, please see:
https://docs.securityonion.net/en/2.3/wazuh.html#usage

**Reply by shapesh** [reply]
Thank you. It works.
